README.md

```diff
@@ -16,7 +16,7 @@ This package is managed with Nuget, in order to update it, follow the next steps
   * `cd src/GitLabApiClient`
   * `rm bin/Release/Apiiro.GitLabApiClient.*.nupkg`
   * `dotnet pack --configuration Release`
-  * `dotnet nuget push bin/Release/Apiiro.GitLabApiClient.*.nupkg --source "github" --skip-duplicate --no-symbols true --api-key <YOUR GITHUB PAT HERE>`
```

Comment:
Note to reviewer: no need for api-key, also boolean flags were changed to either exist or not without a `true` value

Reply:
IIRC I added it because it didn't work for me without adding in the `--api-key`.
No idea why.
I'm in favour of keeping it, maybe a line underneath it like "If you're having authentication issues use this" or something.